### PR TITLE
[XPipeline]Reorder Edit > Rendering

### PR DESCRIPTION
**Pull request by alex-vazquez**
### Purpose of this PR
Reordering SRP related menus from the UX done by @MaddalenaUnity  and @TheoWong-pixel on this [document](https://docs.google.com/spreadsheets/d/14OzGtvhjp_nK-8LAeVlOMLQeyAf2OoZW5Ph519PWv40/edit#gid=195881870)

### **For now this PR is just for Edit > Rendering**

**This PR only affects this menus:**
##### Root
image

##### Lights:
image

##### Decal Layers
image

##### Materials
image

#### Core
- Moved Edit/Render Pipeline/Generate Shader Includes to Edit/Rendering/Generate Shader Includes

#### HDRP
- Moved Edit/Render Pipeline/HD Render Pipeline/Render Selected Camera to log Exr to Edit/Rendering/Render Selected HDRP Camera to log Exr
- Moved Edit/Render Pipeline/HD Render Pipeline/Export Sky to Image to Edit/Rendering/Export HDRP Sky to Image
- Moved Edit/Render Pipeline/HD Render Pipeline/Check Scene Content for Ray Tracing to Edit/Rendering/Check Scene Content for HDRP Ray Tracing
- Moved Edit/Render Pipeline/HD Render Pipeline/Upgrade from Builtin pipeline/Upgrade Project Materials to High Definition Materials to Edit/Rendering/Materials/Convert All Built-in Materials to HDRP"
- Moved Edit/Render Pipeline/HD Render Pipeline/Upgrade from Builtin pipeline/Upgrade Selected Materials to High Definition Materials to Edit/Rendering/Materials/Convert Selected Built-in Materials to HDRP
- Moved Edit/Render Pipeline/HD Render Pipeline/Upgrade from Builtin pipeline/Upgrade Scene Terrains to High Definition Terrains to Edit/Rendering/Materials/Convert Scene Terrains to HDRP Terrains

#### URP
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Project Materials to 2D Renderer Materials to Edit/Rendering/Materials/Convert Project Materials to UniversalRP 2D Renderer
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Scene Materials to 2D Renderer Materials to Edit/Rendering/Materials/Convert Scene Materials to UniversalRP 2D Renderer
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Project Parametric Lights to Freeform to Edit/Rendering/Lights/Convert Project Parametric Lights to Freeform
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Scene Parametric Lights to Freeform to Edit/Rendering/Lights/Convert Scene Parametric Lights to Freeform
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Project Materials to UniversalRP Materials to Edit/Rendering/Materials/Convert Project Materials to UniversalRP
- Moved Edit/Render Pipeline/Universal Render Pipeline/Upgrade Selected Materials to UniversalRP Materials to Edit/Rendering/Materials/Convert Selected Materials to UniversalRP

#### VFX

image

- Moved Edit/Visual Effects//Rebuild And Save All Visual Effect Graphs to Edit/VFX//Rebuild And Save All Visual Effect Graphs

---
### Testing status
Tests that the menus are placed as UX designed

### CODE REVIEW

**Review by JordanL8 — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
@MaddalenaUnity  / @theo-pnv . The Lights scripts 'convert project paramtertics ligth to freeform'  and the others is URP specfic.
The decal layer one is HDRP specific. I guess to be able to handle cross pipeline we should make a distinction about what work for which pipeline. Here we will not be able to know which script have effects. Thought?

**Comment by TheoWong-pixel** [comment]
> @MaddalenaUnity / @theo-pnv . The Lights scripts 'convert project parametric light to freeform' and the others is URP specific.
> The decal layer one is HDRP specific. I guess to be able to handle cross pipeline we should make a distinction about what work for which pipeline. Here we will not be able to know which script have effects. Thought?

Ah yes this is a good point.  We saw that materials mention the RP but not the other ones. So then perhaps we should rename those converters to 
- Convert URP Project Parametric Lights to Freeform
- Convert URP Scene Parametric Lights to Freeform
- Add HDRP Decal Layer Defaults to Selected Mesh Renderer and Terrains 
- Add HDDRP Decal Layer Defaults to  Loaded Mesh Renderer and Terrains 

Also can we rename "Rebuild  And Save All Visual Effect Graphs" to Rebuild and Save All VFX Graphs" please.

**Review by julienf-unity — APPROVED** [review]

**Comment by TheoWong-pixel** [comment]
Sorry! i didnt realise that here all the references to URP is UniversalRP. can we make sure they are all named URP please. :)

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs [hunk 1]
```diff
@@ -138,10 +138,10 @@ public class GeneralSection
...
-            public const string probeVolumeNotEnabled = "Warning: Probe Volumes are not enabled.\nIt is an highly experimental feature and disabled by default for this reason.\nIt's functionality is subject to breaking changes and whole sale removal.\nIt is not recommended for use outside of for providing feedback.\nIt should not be used in production.\nTo enable, set:\nEnableProbeVolumes = 1\ninside of ShaderConfig.cs\and inside of the editor run:\nEdit->Render Pipeline->Generate Shader Includes\nProbe Volumes feature must also be enabled here.";
+            public const string probeVolumeNotEnabled = "Warning: Probe Volumes are not enabled.\nIt is an highly experimental feature and disabled by default for this reason.\nIt's functionality is subject to breaking changes and whole sale removal.\nIt is not recommended for use outside of for providing feedback.\nIt should not be used in production.\nTo enable, set:\nEnableProbeVolumes = 1\ninside of ShaderConfig.cs\and inside of the editor run:\nEdit->Rendering->Generate Shader Includes\nProbe Volumes feature must also be enabled here.";
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs:144]: I see some times there is `>` to seperate menu and sub menu and some times `->`. I think we must use `>` always. @JordanL8 can you confirm this?

File: com.unity.visualeffectgraph/CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [12.0.0] - 2021-01-11
 
-Version Updated
-The version number for this package has increased due to a version update of a related graphics package.
+### Moved
+- Moved Edit/Visual Effects//Rebuild And Save All VFX Graphs to Edit/VFX//Rebuild And Save All Visual Effect Graphs
```
Comment (RSlysz) [inline com.unity.visualeffectgraph/CHANGELOG.md:10]: Same here, it should be `>` insted of `/` or `//` if I understand right.

**Review by RSlysz — APPROVED** [review]
Pre approved but check the documentation separator for menu and sub menu

**Review by martint-unity — COMMENTED** [review]
same feedback as Remi

**Review by martint-unity — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs (hunk 1 above)
Comment (JordanL8) [reply com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs]: Yeah, you're right there. Always **>**

**Review by oleks-k — APPROVED** [review]
Looks good.

**Review by JulienIgnace-Unity — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Review by iM0ve — CHANGES_REQUESTED** [review]
Seems that PR does not compile due to a missing semicolon

image

**Review by iM0ve — APPROVED** [review]
Updated my review

**Pull request merged**